packages/cxl-ui/scss/cxl-credential.scss

```diff
@@ -0,0 +1,132 @@
+:host {
+  display: flex;
+  margin: auto;
+  text-align: start;
+  color: initial;
+}
+
+.container {
+  position: relative;
+  z-index: 0;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 1464px;
+  height: 1120px;
+  padding: var(--lumo-space-xl);
+  overflow: hidden;
+  font-family: var(--lumo-font-family);
+  font-size: var(--lumo-font-size-m);
+  font-weight: 500;
+  color: var(--lumo-primary-contrast-color);
+  text-transform: uppercase;
+  background-color: #d61f2c;
+
+  &[invisible] {
+    visibility: hidden;
+  }
+
+  * {
+    z-index: 2;
+  }
+
+  #circles {
+    position: absolute;
+    top: 0;
+    left: 520px;
+    z-index: 1;
+    width: 100%;
+    height: 100%;
+
+    svg {
+      position: absolute;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      margin: auto;
+    }
+  }
+
+  #cxl-logo {
+    width: 128px;
+    height: 54px;
+
+    --lumo-primary-color: #fff;
+  }
+
+  .thin {
+    font-size: 20px;
+    font-weight: 200;
+    text-transform: lowercase;
+    letter-spacing: 2px;
+  }
+
+  #header {
+    .description {
+      font-size: 20px;
+      font-weight: 900;
+      line-height: 18.84px;
+      color: #010101;
+    }
+
+    .title {
+      font-size: 102.26px;
+      font-weight: 900;
+      margin-block: 0;
+      line-height: 96.33px;
+    }
+  }
+
+  #body {
+    width: 982px;
+    font-size: 45px;
+    font-weight: 900;
+    line-height: 39.96px;
+    color: #010101;
+
+    .highlight {
+      color: var(--lumo-primary-contrast-color);
+    }
+
+    .nowrap {
+      white-space: nowrap;
+    }
+  }
+
+  #footer {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+    height: 353px;
+
+    div.code {
+      text-align: end;
+    }
+  }
+}
+
+:host([type="minidegree"]), :host([theme='dark']) {
+  .container {
+    background-color: #000;
```

Comment:
No `var()` for this value?

Reply:
Both white and black have no fixed variables that keep their values between light and dark themes. The closest would be `--lumo-tint` for white, which is slightly different (#f5f9ff), and for black it would be `--lumo-shade` (#1a1a1a) when in light theme and `--lumo-base-color` (#1a1a1a) when in dark theme. I believe that's confusing and defeats the purpose. Since we just want pure white and pure black in those cases, in both themes, maybe the absolute values make more sense. What do you think?

Reply:
Live site uses `--wp--preset--color--black`. We could define this in the story perhaps?

Reply:
> As far as I see `--lumo-base-color` is `#fff`
> 
> With black indeed I don't see relevant var.

minidegree uses `theme=dark`, and that changes `--lumo-base-color` to `#1a1a1a`. I chose to tie the certificate type to theme because it automatically takes care of most important differences. But In the long term, that might be a problem if we add a dark theme to the whole site. Should I implement the minidegree design independent of lumo theme? That might also make @anoblet suggestion not necessary in the end.

Reply:
Let's leave it for now.